modules/fishnet/src/main/UciToPgn.scala

```diff
@@ -5,14 +5,15 @@ import cats.data.Validated.valid
 import cats.syntax.all.*
 import chess.format.pgn.{ Dumper, SanStr }
 import chess.format.Uci
-import chess.{ Ply, Drop, Move, Replay, Situation }
+import chess.{ Ply, Drop, Move, MoveOrDrop, Replay, Situation }
 import chess.MoveOrDrop.*
 
 import lila.analyse.{ Analysis, Info }
 import lila.base.LilaException
 
-// convert variations from UCI to PGN.
-// also drops extra variations
+// Even though Info.variation is a List[SanStr]
+// When we receive them from Fishnet clients it's actually a list of UCI moves.
+// This converts them to San format. drops extra variations
 private object UciToPgn:
```

Comment:
This is actually converting Info.variation from List[Uci] => List[SanStr], so maybe we need to change the object name.